acme_tiny.py

```diff
@@ -136,8 +135,8 @@ def _poll_until_not(url, pending_statuses, err_msg):
             wellknown_file.write(keyauthorization)
 
         # check that the file is in place
+        wellknown_url = "http://{0}/.well-known/acme-challenge/{1}".format(domain, token)
```

Comment:
According to old code, exception theoretically may happen in `.format()`. In this case code in the `except` block will use undefined variable.

Reply:
More specifically, it could raise a `ValueError`, which would be mistakenly caught.